src/index.js

```diff
@@ -1,10 +1,29 @@
 import './style.css';
 import { showMovies } from './modules/popullateMovie';
 import { showTvShows } from './modules/populateTvshows';
+import {
+  homeLinks, tvLinks, mainContainer, tvCardContainer, cardContainer,
+} from './modules/domElements';
 
 const component = () => {
   showMovies();
   showTvShows();
 };
 
-document.addEventListener('DOMContentLoaded', component);
\ No newline at end of file
+document.addEventListener('DOMContentLoaded', component);
+
+homeLinks.addEventListener('click', () => {
+  tvCardContainer.classList.add('hide');
+  tvLinks.classList.remove('active');
+  homeLinks.classList.add('active');
+  mainContainer.classList.remove('hide');
```

Comment:
- [ ] When I'm trying to drop a comment to a movie, The comment model closes and the comment doesn't get added. I would suggest to fix this issue and the comment functionality should work as per the requirements.